pages/profile/giftfund/index.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { ReactElement } from 'react';
+import UserLayout from '../../../src/features/common/Layout/UserLayout/UserLayout';
+import Head from 'next/head';
+import i18next from '../../../i18n';
+import GiftFunds from '../../../src/features/user/GiftFunds';
+
+const { useTranslation } = i18next;
+
+interface Props {}
+export default function Register({}: Props): ReactElement {
+  const { t } = useTranslation('me');
+  return (
+    <UserLayout>
+      <Head>
+        <title>{t('giftFund')}</title>
+      </Head>
+      <GiftFunds />
```

Comment:
Perhaps this TS warning should be addressed.

```'GiftFunds' cannot be used as a JSX component.
  Its return type 'false | Element' is not a valid JSX element.
    Type 'boolean' is not assignable to type 'ReactElement<any, any>'.```

Reply:
Addressed here - https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1601